torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -1433,23 +1432,23 @@ class StableIValueBoxedKernel : public c10::OperatorKernel {
     const auto num_returns = schema.returns().size();
     const auto num_arguments = schema.arguments().size();
 
-    std::vector<StableIValue> ministack(std::max(num_arguments, num_returns));
+    std::unique_ptr<StableIValue[]> ministack(new StableIValue[std::max(num_arguments, num_returns)]);
 
     for (const auto idx : c10::irange(num_arguments)) {
       const auto ministack_idx = num_arguments - idx - 1;
       const c10::TypePtr& arg_type = schema.arguments()[ministack_idx].type();
-      ministack[ministack_idx] = from_ivalue(arg_type, torch::jit::pop(stack));
+      ministack.get()[ministack_idx] = from_ivalue(arg_type, torch::jit::pop(stack));
```

Comment:
nit: you don't need to change this line because of https://en.cppreference.com/w/cpp/memory/unique_ptr/operator_at